support/package_modsecurity

```diff
@@ -17,11 +17,11 @@ source "$basedir/lib/deb.lib.sh"
 pkgs_url="https://mirrors.edge.kernel.org/ubuntu/pool"
 
 # Tool required to upload nginx build artifacts
-which aws >/dev/null || apt-get install awscli
+which aws >/dev/null || apt-get install --yes awscli
 # Tool required to unarchived .deb data.tar.zst archives (started with ubuntu 22.04)
-which zstd >/dev/null || apt-get install zstd
-# Tool required for building modsecurity
-which aclocal >/dev/null || apt-get install automake libssl-dev libpcre3-dev libxml2-dev libcurl4-openssl-dev
```

Comment:
I don't understand why there is a condition on the presence of `aclocal` to install the libraries. I actually always need the libraries.